segments-guide.md

```diff
@@ -0,0 +1,327 @@
+
+```
+
+{
+   "content":[
+      {
+         "id":"s300000022_591a105ce4b0fc8647cec9ae",
+         "name":"non-oberon segment",
+         "description":"non-oberon segment",
+         "rsid":"obue.analytics.spa",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5924b570e4b0b06b9cc6e74b",
+         "name":"LONDON",
+         "description":"",
+         "rsid":"obue.appservicesprod",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fe2a19678905e58c6f2b6",
+         "name":"test aam",
+         "description":"test aam",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fee889678905e58c6f2c3",
+         "name":"test aam 2",
+         "description":"test aam2",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fef419678905e58c6f2c5",
+         "name":"test aam 3",
+         "description":"test aam 3",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5bb7c94e80f0073611afb35c",
+         "name":"test create",
+         "description":"",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      }
+   ],
+   "totalElements":6,
+   "firstPage":true,
+   "numberOfElements":6,
+   "totalPages":1,
+   "lastPage":true,
+   "sort":null,
+   "size":10,
+   "number":0
+}
+
+```
+
+## Retrieving a Single Segment
+
+You can retrieve segments individually if the segment `id` is known. You can get the segment `id` from  report descriptions or other Analytics screens.
+
+### Example Request
+
+The following example requests data on the segment `s300000022_5bb7c94e80f0073611afb35c` with company ID `obueng0`:
+
+```
+
+curl -X GET "https://analytics.adobe.io/api/obueng0/segments/s300000022_5bb7c94e80f0073611afb35c?locale=en_US" -H "x-api-key: {OAUTHTOKEN}" -H "x-proxy-global-company-id: obueng0" -H "Authorization: Bearer {ACCESSTOKEN}" -H "Accept: application/json" -H "Content-Type: application/json"
+
+```
+
+### Example Response
+
+The following example shows the response data for the request on segment `s300000022_5bb7c94e80f0073611afb35c` with company ID `obueng0`:
+
+```
+
+{
+  "id": "s300000022_5bb7c94e80f0073611afb35c",
+  "name": "test create",
+  "description": "",
+  "rsid": "sistr2.beta",
+  "owner": {
+    "id": 596983
+  }
+}
+
+```
+
+## Deleting a Segment
+
+If you delete a segment, it is hidden from all users in all menus. Additionally, it is no longer returned when retrieving multiple segments in an API call. However, reports and other sources that reference the deleted segment are still able to use it. You can also continue to request it from the single segment API endpoint.
```

Comment:
+1

Reply:
Sorry, I'm missing what should be edited?

Reply:
I'm just saying that it is well done:)

Reply:
Thanks.